website/success.html

```diff
@@ -0,0 +1,98 @@
+<!doctype html>
```

Comment:
Feedback regarding styling is the same as for the previous files.

The layout looks great. We also need to enable manual grading in case CV algorithms fail to produce a good grade. Let's have button's next to each value that the CV algorithms return so that the users can modify them in case they are wrong. So for example if the CV algorithms gives a high grade to terrible packaging, allow the user to modify that. Or if it misidentifies the retailer, allow the user to correct it.